js/interactive-examples.js

```diff
@@ -0,0 +1,62 @@
+import phpBinary from "/js/php-web.mjs";
+
+function createOutput(output) {
+  const container = document.createElement('div');
+  container.classList.add('screen', 'example-contents');
+  const title = document.createElement('p');
+  title.innerText = 'Output: ';
+  container.appendChild(title)
+  const div = document.createElement('div');
+  div.classList.add('examplescode');
+  container.appendChild(div);
+  const pre = document.createElement('pre');
+  pre.classList.add('examplescode');
+  pre.innerText = output;
+  div.appendChild(pre)
+  return container;
+}
+
+async function main() {
+  const buffer = [];
+  const { ccall } = await phpBinary({
+    print(data) {
+      if (!data) {
+        return;
+      }
+
+      if (buffer.length) {
+        buffer.push('\n');
+      }
+      buffer.push(data);
+    }
+  })
+
+  console.log("PHP wasm %s loaded.", ccall("phpw_exec", "string", ["string"], ["phpversion();"]));
+  let lastOutput = null
+
+  document.querySelectorAll('.example').forEach((example) => {
+    const button = document.createElement('button');
+    button.setAttribute('type', 'button')
+    const phpcode = example.querySelector('.phpcode');
+
+    const code = phpcode.querySelector('pre code')
```

Comment:
```suggestion
    const code = phpcode.querySelector('code')
```

I think `pre` is not needed here, is it?

On the preview site https://web-php-pr-1097.preview.thephp.foundation/manual/en/function.var-dump.php
I see the following error in the console:
![image](https://github.com/user-attachments/assets/238c25a4-530e-4782-85e4-a3cadff1cd61)

Reply:
weird I have a different HTML on the local version but it works